recon/src/eventid.rs

```diff
@@ -0,0 +1,175 @@
+use cbor::Encoder;
+use cid::Cid;
+use multihash::{Hasher, Sha2_256};
+/// EventId generates EventIDs from event data.
+///
+/// varint(0xce) + // streamid, 1 byte
+/// varint(0x30) + // dag-cbor, 1 byte
```

Comment:
Why 0x30? Also, I guess it should no longer say dag-cbor?

Reply:
When we moved from `dag-cbor` to `multicodec` I changed the number to `0x30` the multicode for `multicodec` itself.

I have no preference between
`0x30` multicodec
`0x05` next available
`0x7c` for cip-124 (0x7c)

`0x00` - `0x04` are taken
`0x80` - u64 are more than one byte.
so any number `0x05` - `0x7f` is fine by me 🤷‍♂️

Reply:
What do you mean that we moved "to multicodec"? We use `dag-cbor` because that's how we encoded the payload. I don't understand what `multicodec` signifies here

Reply:
we picked `0x05`